packages/cardano-services/src/StakePool/DbSyncStakePoolProvider/DbSyncStakePoolProvider.ts

```diff
@@ -26,10 +26,20 @@ import { toStakePoolResults } from './mappers';
  * Properties that are need to create DbSyncStakePoolProvider
  */
 export interface StakePoolProviderProps {
+  /**
+   * Enables useBlockfrost search parameters.
```

Comment:
Can you elaborate on this? It's still not clear why we need two params

Reply:
OK, I understand now. Having this exposed in the search params to facilitate testing is a code smell. Can we not instead just load two `ProviderServer`s, one with Blockfrost cache enabled, the other without, then compare?